tests/test_branch_gen_answers.py

```diff
@@ -0,0 +1,10 @@
+# First Party
+from instructlab.eval.mt_bench import MTBenchBranchEvaluator
+
+mt_bench_branch = MTBenchBranchEvaluator(
+    "instructlab/granite-7b-lab",
+    "instructlab/granite-7b-lab",
+    "../taxonomy",
```

Comment:
@danmcp maybe I'm missing context or another PR that's being worked on, do these tests run in an automated fashion anywhere yet? 

The setup for this test would include cloning a taxonomy repo and then passing that path in right?

Reply:
Not automated yet.  Instructions are in the README for now.